addons/tagging/functions/fnc_createTag.sqf

```diff
@@ -19,15 +19,21 @@
 
 #include "script_component.hpp"
 
-params ["_tagPosASL", "_vectorDirAndUp", "_color", "_object"];
-TRACE_4("createTag:", _tagPosASL, _vectorDirAndUp, _color, _object);
+params ["_tagPosASL", "_vectorDirAndUp", "_colorTexture", "_object"];
+TRACE_4("createTag:",_tagPosASL,_vectorDirAndUp,_colorTexture,_object);
 
-if !((toLower _color) in ["black", "red", "green", "blue"]) exitWith {
-    ACE_LOGERROR_1("%1 is not a valid tag colour.", _color);
+private _customTexture = [true, false] select (_colorTexture find ".paa" == -1);
+
+if (!_customTexture && {!((toLower _colorTexture) in ["black", "red", "green", "blue"])}) exitWith {
+    ACE_LOGERROR_1("%1 is not a valid tag colour.",_colorTexture);
 };
 
 private _tag = "UserTexture1m_F" createVehicle [0,0,0];
-_tag setObjectTextureGlobal [0, '\z\ace\addons\tagging\UI\tags\' + _color + '\' + str (floor (random 3)) + '.paa'];
+if (!_customTexture) then {
+    _tag setObjectTextureGlobal [0, "\z\ace\addons\tagging\UI\tags\" + _colorTexture + "\" + str (floor (random 3)) + ".paa"];
```

Comment:
If we want to add this type of customization, I'd recommend changing the `_colorTexture` variable ot `_texture`, and moving all the color logic out of `fnc_createTag` and `fnc_tagDirection` and directly into the ACE_Action statement. Now it's the time to do it, before the PR that add tagging events is merged.